app/code/Magento/PaypalGraphQl/etc/schema.graphqls

```diff
@@ -51,6 +51,7 @@ input PaymentMethodInput {
     payflow_link: PayflowLinkInput @doc(description:"Required input for PayPal Payflow Link and Payments Advanced payments")
     payflowpro: PayflowProInput @doc(description: "Required input type for PayPal Payflow Pro and Payment Pro payments")
     hosted_pro: HostedProInput @doc(description:"Required input for PayPal Hosted pro payments")
+    payflowpro_cc_vault: VaultTokenInput
```

Comment:
Add a description for payflowpro_cc_vault

Reply:
**Done**